menas/src/main/scala/za/co/absa/enceladus/menas/controllers/LandingPageController.scala

```diff
@@ -48,4 +59,14 @@ class LandingPageController @Autowired() (datasetRepository: DatasetMongoReposit
       todaysStats <- runsService.getTodaysRunsStatistics()
     } yield LandingPageInformation(dsCount, mtCount, schemaCount, runCount, todaysStats)
   }
+
+  @Async
+  @Scheduled(fixedDelay = 900000)
+  def scheduledTest(): CompletableFuture[_] = {
```

Comment:
```suggestion
  @Scheduled(initialDelay = 1000, fixedDelay = 900000)
  def scheduledLandingPageStatsRecalc(): CompletableFuture[_] = {
```
I hope I understand the `@Scheduled` annotation correctly, but it seems to me that we are risking that in the first 15 minutes there is no landing page info cached.

Reply:
Good point, I also wondered! :) Thanks